APCAppCore/APCAppCore/Library/Categories/NewDataCollector/Collectors/APCHealthKitBackgroundDataCollector.m

```diff
@@ -90,7 +90,7 @@ - (void)start
         
         [self.healthStore requestAuthorizationToShareTypes:nil
                                                  readTypes:readTypes
-                                                completion:nil];
+                                                completion:^(BOOL success __unused, NSError * _Nullable error __unused) {}];
```

Comment:
yes, sort of...

HealthKit has a bug where this completion is improperly defined as nullable so it isn't recognized by the compiler as such. The _Nullable flags were added for Swift.